src/hooks/useFilteredCommands.ts

```diff
@@ -61,8 +61,8 @@ const useFilteredCommands = (
         const chars = search.toLowerCase().split('')
 
         return (
-          // include commands with at least one included char and no more than three chars non-matching chars
-          // fuzzy matching will prioritize the best commands
+          // include shortcuts with at least one included char and no more than three chars non-matching chars
+          // fuzzy matching will prioritize the best shortcuts
```

Comment:
This should be "commands" or "command labels", since it is referring to the command labels, not the command shortcuts.